include/exiv2/datasets.hpp

```diff
@@ -187,7 +187,7 @@ namespace Exiv2 {
          */
         static const char* dataSetDesc(uint16_t number, uint16_t recordId);
         /*!
-          @brief Return the photohsop name of a given dataset.
+          @brief Return the photoshop name of a given dataset.
```

Comment:
Photoshop should also be capitalized (as a brand name)? See also line 193 below (and look for other instances)?